db/patch-group-table-change-weight-column-to-int-.sql

```diff
@@ -0,0 +1,52 @@
+-- change column weight from boolean to integer in group and group_monitor tables
+PRAGMA foreign_keys=off;
+BEGIN TRANSACTION;
+
+CREATE TABLE IF NOT EXISTS [new_group]
+(
+    id           INTEGER      not null
+        constraint group_pk
+            primary key autoincrement,
+    name         VARCHAR(255) not null,
+    created_date DATETIME              default (DATETIME('now')) not null,
+    public       BOOLEAN               default 0 not null,
+    active       BOOLEAN               default 1 not null,
+    weight       INTEGER      NOT NULL DEFAULT 1000,
+    status_page_id INTEGER
+);
+
+CREATE TABLE [new_monitor_group]
+(
+    [id]         INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    [monitor_id] INTEGER                           NOT NULL REFERENCES [monitor] ([id]) ON DELETE CASCADE ON UPDATE CASCADE,
+    [group_id]   INTEGER                           NOT NULL REFERENCES [group] ([id]) ON DELETE CASCADE ON UPDATE CASCADE,
+    weight INTEGER NOT NULL DEFAULT 1000
+);
+
+CREATE INDEX [fk]
+    ON [monitor_group] (
+    [monitor_id],
+    [group_id]);
```

Comment:
I think this is more of a hassle here. Mainly, it's unnecessary because it works with an old table.

Leaving only the bottom should be sufficient.